book/section3/didcomm.md

```diff
@@ -2,11 +2,92 @@
 
-[DIDComm Messaging](https://identity.foundation/didcomm-messaging/spec/) messages are encrypted JSON Web Message ([JWM](https://datatracker.ietf.org/doc/html/draft-looker-jwm-01)) envelopes. They have a standard structure including headers, a body, and optional attachments, which are encrypted and signed to ensure security and integrity.
+At its core, DIDComm serves a fundamental purpose: enabling secure, private, and authenticated communication between entities identified by DIDs, regardless of their underlying infrastructure or technology stack. This capability transforms DIDs from static identifiers into dynamic communication endpoints capable of engaging in rich interactions.
 
-For details on how messages are sent and received in Identus, see [Mediators](../section4/mediator.md)
+Within the SSI technology stack, DIDComm occupies a pivotal middle layer. It sits above the foundational DID layer (which provides identifiers and cryptographic material) and below the application-specific protocols that define particular interactions like credential exchange. This positioning makes DIDComm the connective tissue of the SSI ecosystem, enabling all higher-level identity interactions.
+
+## Key Characteristics
+
+DIDComm's design incorporates several distinctive characteristics that make it particularly well-suited for decentralized identity communications:
+
+**Transport agnosticism** allows DIDComm messages to travel over virtually any communication channel—HTTP(S), WebSockets, Bluetooth, NFC, mesh networks or even QR codes. This flexibility means that identity interactions aren't tied to specific network infrastructures, enabling truly peer-to-peer communications even in offline or intermittent connectivity scenarios.
+
+**End-to-end encryption** ensures that message contents remain confidential between the sender and intended recipient(s), even when traversing untrusted networks or intermediaries. Using public key cryptography derived from DIDs, DIDComm creates secure communication channels that protect sensitive identity information.
+
+**Authentication and trust** are inherent in DIDComm, as every message can be cryptographically verified to have originated from a specific DID. This authentication happens without centralized authorities, allowing parties to establish trust directly with one another.
+
+**Asynchronous capabilities** enable communications that don't require both parties to be online simultaneously. Messages can be queued, stored, and forwarded until they reach their destination, making DIDComm suitable for a wide range of real-world scenarios where continuous connectivity isn't guaranteed.
+
+**Message routing and forwarding** mechanisms allow communications to traverse complex paths, including through mediators and relays, while maintaining end-to-end security. This capability is essential for entities behind firewalls or NATs, or for preserving additional privacy by obscuring network-level metadata.
```

Comment:
This is a good, helpful breakdown of the characteristics and benefits of DIDComm. 👍🏼